src/nprintml/cli.py

```diff
@@ -0,0 +1,22 @@
+import argparse
+
+# import argparse_formatter  # FIXME
+
+
+def execute(argv=None, **parser_kwargs):
+    parser = argparse.ArgumentParser(
+        description='DESCRIPTION GOES HERE',
+
+        # TODO: It's assumed that we'll want fancy formatting;
+        # TODO: but, if that's not the case let's get rid of this:
+        # formatter_class=argparse_formatter.ParagraphFormatter,
```

Comment:
`argparse` is annoyingly simplistic with its formatting. for any reasonably-well documented interface, you'll likely at least want this `ParagraphFormatter` extension, (_s.t._ it doesn't snip all your newlines). but, I thought I shouldn't make _too many_ assumptions.

(see for example `manage.py:Version` – without this extension, the two paragraphs of its description are merged into one line.)

Reply:
I think that with our feature aggregation vision we'll need well-formatted documents, this makes sense.